cluster/saltbase/salt/rescheduler/rescheduler.manifest

```diff
@@ -28,8 +28,8 @@ spec:
     # TODO: split this out into args when we no longer need to pipe stdout to a file #6428
     - sh
     - -c
-    - '/rescheduler --running-in-cluster=false 1>>/var/log/rescheduler.log 2>&1'
+    - '/rescheduler --running-in-cluster=false --log_dir=/var/log/rescheduler --logtostderr=false'
```

Comment:
after the fix https://github.com/kubernetes/contrib/commit/07514af5130851992cedff5b60a4aeb64970ab88, we enable the logrotate and currently, rescheduler only support use --log_dir to write log files in a directory, see rescheduler --help output:
Usage of rescheduler: rescheduler --running-in-cluster=true:
      --alsologtostderr                  log to standard error as well as files
      --housekeeping-interval duration   How often rescheduler takes actions. (default 10s)
      --initial-delay duration           How long should rescheduler wait after start to make sure
		 all critical addons had a chance to start. (default 2m0s)
      --kube-api-content-type string     Content type of requests sent to apiserver. (default "application/vnd.kubernetes.protobuf")
      --listen-address string            Address to listen on for serving prometheus metrics (default "localhost:9235")
      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                   If non-empty, write log files in this directory
      --logtostderr                      log to standard error instead of files (default true)
      --min-replica-count int            Minimum number or replicas that a replica set or replication controller should have to allow their pods deletion in scale down
      --pod-scheduled-timeout duration   How long should rescheduler wait for critical pod to be scheduled
		 after evicting pods to make a spot for it. (default 10m0s)
      --running-in-cluster               Optional, if this controller is running in a kubernetes cluster, use the
		 pod secrets for creating a Kubernetes client. (default true)
      --skip-nodes-with-local-storage    If true cluster autoscaler will never delete nodes with pods with local storage, e.g. EmptyDir or HostPath (default true)
      --skip-nodes-with-system-pods      If true cluster autoscaler will never delete nodes with pods from kube-system (except for DeamonSet or mirror pods) (default true)
      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
      --system-namespace string          Namespace to watch for critical addons. (default "kube-system")
  -v, --v Level                          log level for V logs
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging

so we may need to update the salt file to not to create the rescheduler.log and place the log file under /var/log/rescheduler.